### ISSUE TITLE
404 while redirect in FluentGridFieldExtension::actionComplete 

### ISSUE THREAD

**Issue by bumbus** [tractorcow-farm/silverstripe-fluent#614]
This happens while "delete from all locales" in a Gridfield located in `SiteTree::getCMSFields`

https://github.com/tractorcow-farm/silverstripe-fluent/blob/89278929e4f8509c2e10cbafd3d9c95ef8b21ac7/src/Extension/FluentGridFieldExtension.php#L69

This produces `admin/pages/edit/` out of something like this `admin/pages/edit/EditForm/1/field/Sections/item/7/ItemEditForm` but it needs to be something like this `admin/pages/edit/show/1`

**Comment by tractorcow** [comment]
Good point, I think I have an idea about what's causing this. I'll investigate.
